hyprbars/barDeco.cpp

```diff
@@ -157,9 +176,11 @@ void CHyprBar::renderBarTitle(const Vector2D& bufferSize) {
 }
 
 void CHyprBar::renderBarButtons(const Vector2D& bufferSize) {
+    static const int CLOSE_COLOR = HyprlandAPI::getConfigValue(PHANDLE, "plugin:hyprbars:button_close_color")->intValue;
+    static const int MAX_COLOR   = HyprlandAPI::getConfigValue(PHANDLE, "plugin:hyprbars:button_max_color")->intValue;
```

Comment:
Incorrect, has to be a pointer, otherwise it's unmodifiable at runtime.

`static auto* const PMAXCOLOR = &HyprlandAPI::getConfigValue(PHANDLE, "plugin:hyprbars:button_max_color")->intValue;`

same for close color.

\+ No snake.

Reply:
thanks for the explanation, done.